src/SAWScript/Prover/MRSolver/Monad.hs

```diff
@@ -252,6 +246,8 @@ asEither _ = Nothing
 -- | A map from 'Term's to 'DataTypeAssump's over that term
 type DataTypeAssumps = HashMap Term DataTypeAssump
 
+-- FIXME HERE NOW: remove preconditions from MREnv
+
```

Comment:
Was this resolved? If so this comment should probably be removed.

Reply:
Whoops, yep, should be removed